dbbackup/management/commands/dbrestore.py

```diff
@@ -77,7 +77,8 @@ def restore_backup(self):
         if not self.filepath:
             self.log("  Finding latest backup", 1)
             filepaths = self.storage.list_directory()
-            filepaths = [f for f in filepaths if f.endswith('.' + self.backup_extension)]
+            # TODO: It is a bad filter
+            # filepaths = [f for f in filepaths if f.endswith('.' + self.backup_extension)]
```

Comment:
> We should be able to find which are db-backups and find last backup by a real meta data, not the last of list.

Yes, it's better to use the consistent naming of files and their possible meta data attributes such as the `create_time` attribute that you mention.

Does the commented out filter give any problems? Is it the `gz` end that's causing this?